src/test/java/duke/EventTest.java

```diff
@@ -0,0 +1,18 @@
+package java.duke;
+
+import duke.DukeEmptyDescriptionException;
+import duke.Event;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+    @Test
+    public void eventTest(){
```

Comment:
You may want to consider changing your test names to something more related to what is being tested. For instance
`sortList_emptyList_exceptionThrown` or `getMember_memberNotFount_nullReturned`